include/baresip.h

```diff
@@ -184,11 +185,13 @@ enum call_event {
 /** Call States */
 enum call_state {
 	CALL_STATE_IDLE = 0,
+	CALL_STATE_ARRIVED,
```

Comment:
please add new values at the end of the enum ... for better binary compat ...

Reply:
okay.

Should we keep `CALL_STATE_UNKNOWN` as last value?